static/js/publisher/form/fileInput.js

```diff
@@ -55,12 +88,85 @@ class FileInput extends React.Component {
     }
   }
 
+  dragOverHandler(e) {
+    e.preventDefault();
+
+    this.setState({
+      isDragging: true,
+      canDrop: false
+    });
+
+    if (e.target === this.holder) {
+      this.setState({
+        canDrop: true,
+        cantDrop: false
```

Comment:
Is this needed? Or do you want just to be double sure that user can drop? ;)

Reply:
Ah, bad rebase conflict artifact, thanks